mtad.yaml

```diff
@@ -30,9 +30,9 @@ resources:
     type: org.cloudfoundry.managed-service
     parameters:
       service: postgresql
-      service-plan: v9.4-container
+      service-plan: v9.4-dev-large
   - name: spring-music-news-external
     type: org.cloudfoundry.user-provided-service
     parameters:
       config:
-        url: # unspecified, as its value is not known on app design time
\ No newline at end of file
+        EXTERNAL_NEWS_URL: trial-i069874-spring-music-news-external.cfapps.sap.hana.ondemand.com/news
```

Comment:
Put this value in an extension descriptor like in https://github.com/nvvalchev/spring-music/blob/master/config.mtaext.